src/containers/scheduleExams/scheduleExams.tsx

```diff
@@ -1,11 +1,12 @@
 import { GridWrapper, Header, WordAccent } from './scheduleWrapper.style';
-import Exam from '@/components/examComponent';
-import { useGroupContext } from '@/common/context/groupContext';
-import { useWeekContext } from '@/common/context/weekContext';
+import Exam from '../../components/examComponent';
+import { useGroupContext } from '../../common/context/groupContext';
+import { useWeekContext } from '../../common/context/weekContext';
 import React, { useEffect, useState } from 'react';
 
 interface Props {
-  getData: (id: number) => Promise<any>
+  getData: (id: string) => Promise<any>
+  contextType?: string
```

Comment:
Can't comment below this line, but on line 31: groupContext?.group.name will result in exception on clean app run.

Reply:
Still not fixed:

<img width="1646" alt="Screenshot 2024-08-04 at 23 17 08" src="https://github.com/user-attachments/assets/768e3082-5588-4321-9a66-1cd6395697fb">